Warmup-1/startHi/test/index.test.ts

```diff
@@ -0,0 +1,25 @@
+/*
+Given a string, return true if the string starts with "hi" and false otherwise.
+
+startHi("hi there") → true
+startHi("hi") → true
+startHi("hello hi") → false
+*/
+
+import { assert } from "chai";
+import { startHi } from "../src";
+
+describe("startHi", () => {
```

Comment:
Can you please add a test for when the string is `Hi there`?

Reply:
Sure, test added.